forest/cli/src/cli/snapshot_cmd.rs

```diff
@@ -53,6 +53,9 @@ pub enum SnapshotCommands {
         /// in default Forest data location.
         #[structopt(short, long)]
         snapshot_dir: Option<PathBuf>,
+        /// Use aria2 for downloading, default is false.
```

Comment:
`aria2` should be a code block or, better yet, a link. Then we can remove it from the dictionary.

Reply:
Fixed